tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/types/intersection/IntersectionTypeTest.java

```diff
@@ -264,6 +264,12 @@ public void testUnsupportedIntersectionNegative() {
         assertEquals(result.getErrorCount(), index);
     }
 
+    @Test
+    public void testErrorIntersectionAccessTest() {
+        CompileResult result = BCompileUtil.compile("test-src/types/intersection/error-intersection-access");
+        assertEquals(result.getErrorCount(), 0);
```

Comment:
Please add an assertion or two for `code` too.

Reply:
You're not calling the function for the assertions to happen?

Reply:
Good catch!